### PR TITLE
Revise glossary definitions for cluster infrastructure & operations

### LINKED ISSUE #18294: Issue with k8s.io/docs/reference/glossary/
**This is a Bug Report**

**Problem:**
`cluster infrastructure` and `cluster operations` are not included in glossary while there are references to them in https://kubernetes.io/docs/setup/#production-environment

**Proposed Solution:**
Add two terms

**Page to Update:**
https://kubernetes.io/docs/reference/glossary/?all=true

### PR DESCRIPTION
**Pull request by sftim**
Fixes #18294 

- Fix metadata typo (`operations` tag should be `operation`)
- Improve wording

### CODE REVIEW

**Comment by bradtopol** [comment]
See inline  comments. 
/assign

**Comment by sftim** [comment]
@bradtopol I _don't_ see any inline comments; is this maybe pending a `COMMIT` :wink:?

**Comment by kbhawkey** [comment]
@bradtopol 👋 , Did you intend to add comments/review this PR? Thanks!

**Comment by kbhawkey** [comment]
Page previews:
https://deploy-preview-18317--kubernetes-io-master-staging.netlify.com/docs/reference/glossary/?all=true#term-cluster-infrastructure

https://deploy-preview-18317--kubernetes-io-master-staging.netlify.com/docs/reference/glossary/?all=true#term-cluster-operations

**Comment by bradtopol** [comment]
@kbhawkey @sftim My comment was I didn't think the phrase "without a long term financial commitment" was needed or appropriate in a definition of cloud infrastructure.   Typically something that mentions elastic allocation of resources or something like that

**Comment by bradtopol** [comment]
@sftim @kbhawkey somehow my initial inline comment never got submitted.  I just did it now. My apologies! :-)

File: content/en/docs/reference/glossary/cluster-infrastructure.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ short_description: >
...
-- operations
+- operation
 ---
-The infrastructure layer provides and maintains VMs, networking, security groups and others.
+ The infrastructure layer provides and maintains VMs, networking, security groups and others.
+
+<!--more-->
+
+A _cloud_ infrastructure lets you use utility computing services to deploy infrastructure
+components, accessible over the internet, and typically without a long term financial
```
[the remark this answers is not in the document]
Comment (kbhawkey) [reply content/en/docs/reference/glossary/cluster-infrastructure.md]: 👍

File: content/en/docs/reference/glossary/cluster-operations.md [hunk 2]
```diff
@@ -4,10 +4,17 @@ id: cluster-operations
...
-- operations
+- operation
 ---
- Activities such as upgrading the clusters, implementing security, storage, ingress, networking, logging and monitoring, and other operations involved in managing a Kubernetes cluster.
+ The work involved in managing a Kubernetes cluster.
+
+<!--more-->
+
+ Examples of cluster operations work include: deploying new Nodes to
+scale the cluster; performing software upgrades; installing security
```
Comment (kbhawkey) [inline content/en/docs/reference/glossary/cluster-operations.md]: @sftim , nit: The original definition divides the work into:
 upgrading clusters (software and hardware) and administering (managing) clusters.
I think implementing logging and monitoring is different from `responding to events`.
Is `implementing security` the same as `installing security`. What do you think?

Comment (sftim) [reply content/en/docs/reference/glossary/cluster-operations.md]: I'll bear that in mind!

**Comment by sftim** [comment]
@bradtopol no worries

**Comment by sftim** [comment]
/retitle [WIP] Fix glossary definitions for cluster infrastructure & operations

**Comment by sftim** [comment]
/retitle Revise glossary definitions for cluster infrastructure & operations

**Pull request merged**